fineract-provider/build.gradle

```diff
@@ -403,7 +404,14 @@ war {
 
 license {
     header rootProject.file('../APACHE_LICENSETEXT.md')
-    excludes(["**/*.html", "**/*.mustache", "**/package-info.java", "**/keystore.jks", "**/swagger-ui/**"])
+    excludes([
```

Comment:
Nice, but in a perfect world better raised in a very small separate PR. (I won't be a PITA and still merge it if you leave this one here though.  But that RAT related change really would be better to separate out.)

Reply:
Ok. Felt the line was getting long with what I added, so decided to break it.

Reply:
Sure, OK.